cmd/mtping/main.go

```diff
@@ -39,17 +39,5 @@ func main() {
 
 	ctx = adapter.WithController(ctx, mtping.NewController)
 	ctx = adapter.WithHAEnabled(ctx)
-
```

Comment:
have we discussed this potential issue, upstream?

Reply:
we could potentially look for a thread upstream.

(I am not blocking the fix here for our product ... FWIW)